.github/scripts/test_behavior/plan.py

```diff
@@ -221,9 +221,10 @@ def generate_language_binding_cases(
 ) -> list[dict[str, str]]:
     cases = unique_cases(cases)
 
-    # Remove hdfs cases for java.
+    # Remove specified services cases for java.
```

Comment:
Please don't touch those files, hdfs_native is pure rust which should be fine on other bindings.

Reply:
Sure 
Due to failures in behaviour tests I was trying a bunch of stuff to figure them out. Could you please help with the failures ?